### DISCUSSION TITLE
Showing Section Seprator in Navbar

### DISCUSSION THREAD

**Discussion by arky** [thuliteio/doks#1272] [Q&A] [answered]
How to show separator pipe `|` between sections of social and control items in navigation bar.  

![image](https://github.com/user-attachments/assets/8976bb53-110f-49b4-8073-0e72acfae54b)

**Comment by jollydodo** [comment, marked as answer]
Replace the existing header with your own header by making a new file in layouts -> partials -> header -> header.html. 
Here you can copy your existing one from the doks-core (located in your npm module folder under Hyas).  Now modify it for the piping. You want to look for '' in this file and add this bit of code `<li class="nav-item d-none d-lg-flex py-1 col-auto">
            <div class="vr h-100 mx-2 text-muted"></div>
          </li>` directly under ` <ul id="socialMenu" class="nav mx-auto flex-row order-lg-4">`

You can look at the example from the 'getdoks' website if this is not clear for you.
Your original file:
https://github.com/gethyas/doks-core/blob/main/layouts/partials/header/header.html (row 243)

modified file with separator :
https://github.com/gethyas/getdoks.org/blob/main/layouts/partials/header/header.html (row 275)

**Reply by arky** [reply]
Thank you @jollydodo
